docs/docs/guide/basics/VersionedUrl.tsx

```diff
@@ -1,8 +1,7 @@
 import React from "react";
-import packageJson from "../../../../@stellar/freighter-api/package.json";
 
 export const VersionedUrl = () => (
   <p>
-    <code>{`<head><script src='https://cdnjs.cloudflare.com/ajax/libs/stellar-freighter-api/${packageJson.version}/index.min.js' /></head>`}</code>
+    <code>{`<head><script src='https://cdnjs.cloudflare.com/ajax/libs/stellar-freighter-api/{version}/index.min.js' /></head>`}</code>
```

Comment:
What's the difference between these two? I couldn't find much on the global `version`.

Reply:
For context, I added the original logic around grabbing the package json file because cdnjs creates a url based on the latest version. So, for ex, to get v3.0.0 of freighter-api, you would need a url of: https://cdnjs.cloudflare.com/ajax/libs/stellar-freighter-api/3.0.0/index.min.js

How does the `version` var accomplish that?

Reply:
Ah yeah, so there isn't a global `version`. I changed it to be consistent with the text instructions [here](https://docs.freighter.app/docs/guide/usingFreighterBrowser) which mentions `{version}` . This is the only place importing this `VersionedUrl` component.

Reply:
Humm maybe we should just put a plain `html` code like it's being done [here](https://github.com/stellar/freighter/blob/master/docs/docs/guide/gettingStarted.md) instead of importing this `VersionedUrl` component. Wdyt?

Something like:

```html
<head><script src="https://cdnjs.cloudflare.com/ajax/libs/stellar-freighter-api/{version}/index.min.js"></script></head>
```

Reply:
Oh, just to give further context: I did originally have it as plain html with the `{version}` var in it, but I got complaints from people in the org that the link should be dynamic and include the latest version so users can just copy and paste without having to insert the version number themself

Reply:
This is the original ticket that caused me to add this: https://github.com/stellar/freighter/issues/829

Reply:
Ah, good to know! Ok, I'll revert my change and suggest another change on the body text

Reply:
Tweaked the text a little bit here: https://github.com/stellar/freighter/pull/1629/commits/7375c2182a0bc5a897097f56186b84b815bc943f

Reply:
Also reverted the `VersionedUrl` change here: https://github.com/stellar/freighter/pull/1629/commits/217eda9119a391419cc064295b68d60aa0747cfb